src/components/getting-started/fileblock.js

```diff
@@ -110,18 +110,13 @@ class FileBlock extends React.Component {
               <i aria-hidden='true' className='fa fa-content-copy' />
             </a>
           </div>
-          <ReactCSSTransitionGroup
-            transitionEnterTimeout={1000}
-            transitionLeaveTimeout={1000}
-            transitionName={'checkmark'}
+          <BaseTransition
+            in={this.state.clicked}
+            timeout={{ enter: 1000, exit: 1000 }}
+            classNames='checkmark'
           >
-            {this.state.clicked ? (
-              <i
-                aria-hidden='true'
-                className='fa fa-done codeblock--checkmark'
-              />
-            ) : null}
-          </ReactCSSTransitionGroup>
+            <i aria-hidden='true' className='fa fa-done codeblock--checkmark' />
```

Comment:
This one seems to not work anymore, at least for me.

They show up on the getting started pages of a cluster, when clicking the clipboard icon: http://localhost:7000/organizations/giantswarm/clusters/bs04o/getting-started/configure/

<img width="782" alt="Screenshot 2019-11-13 at 10 02 35 AM" src="https://user-images.githubusercontent.com/455309/68748528-e3cd4700-05fc-11ea-968e-48722c11f754.png">

Reply:
actually the checkmark became super tiny